### PR TITLE
Game relation schema update

### PR DESCRIPTION
**Pull request by FifthSurprise**
This PR governs all the changes required to allow multiple games of Watch the Skies to run concurrently.

In particular, the PR covers all of the base migrations and relations that need to be set up.

Please note this will cause significant changes to the database.  Data preservation was not a goal so I would recommend wiping out your database and re-seeding with updated seed file.

Now, every table should relate back to a game.  Users will also have to be tied to a game.

If you are working on a controller, you can use the application_controller method `current_game` in order to fetch the game the user is to assigned to.  If the user does not have a game, it defaults to the last game.  However this is behavior that will probably change soon as all users should be assigned to a game or not have access to most of the app.

Other things still in the process (but might be put into this PR) would be user registration, assigning users to games via admin/ upgrading users as SuperAdmin, and a main splash page (since the dashboard itself is no longer is relevant as the main page).  However parts of this PR is a blocker for others and so I'm opening it now for review.

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -26,7 +26,7 @@
     t.datetime "updated_at"
     t.string   "control_message"
     t.string   "activity"
-    t.string   "time_zone",       limit: 255, default: "UTC"
+    t.string   "time_zone",       limit: 255, default: "Pacific Time (US & Canada)"
```
Comment (michaelpelletier) [inline db/schema.rb]: Hmmm. Why this change?

File: db/seeds.rb [hunk 2]
```diff
@@ -18,56 +26,54 @@
...
 
 #An example message for the database. Probably doesn't need to exist in the final version.
-m = Message.create(
-	sender: "UK",
-	recipient: "France",
-	content: "This is an example message",
-	round_number: 0,
-	game_id: g.id
-)
+# m = Message.create(
```
Comment (michaelpelletier) [inline db/seeds.rb]: Is the example message intentionally commented out? At first I assumed yes, but the example news messages are left in a few lines below.

**Comment by michaelpelletier** [comment]
Looks great! Just had two small questions.

File: db/schema.rb (hunk 1 above)
Comment (FifthSurprise) [inline db/schema.rb]: Leftover from un-run migration from @x41farmer that got merged in.  No harm in leaving it there for now until time zones/user preferences is working.

File: db/seeds.rb (hunk 2 above)
Comment (FifthSurprise) [inline db/seeds.rb]: Yes, right now Message does not really do anything.  It was supposed to handle player to alien communications but it's just not utilized.  No point in seeding data.

Comment (michaelpelletier) [inline db/seeds.rb]: Fair point

File: db/schema.rb (hunk 1 above)
Comment (michaelpelletier) [inline db/schema.rb]: Got it. :+1:
